stubs/pika/pika/data.pyi

```diff
@@ -1,6 +1,14 @@
-def encode_short_string(pieces, value): ...
-def decode_short_string(encoded, offset): ...
-def encode_table(pieces, table): ...
-def encode_value(pieces, value): ...
-def decode_table(encoded, offset): ...
-def decode_value(encoded, offset): ...
+from collections.abc import Mapping
+from datetime import datetime
+from decimal import Decimal
+from typing_extensions import TypeAlias
+
+_Value: TypeAlias = str | bytes | bool | int | Decimal | datetime | _ArgumentMapping | list[_Value] | None
+_ArgumentMapping: TypeAlias = Mapping[str, _Value]
+
+def encode_short_string(pieces: list[bytes], value: str | bytes) -> int: ...
+def decode_short_string(encoded: bytes, offset: int) -> tuple[str, int]: ...
+def encode_table(pieces: list[bytes], table: _ArgumentMapping) -> int: ...
+def encode_value(pieces: list[bytes], value: _Value) -> int: ...
+def decode_table(encoded: bytes, offset: int) -> dict[str, _Value]: ...
```

Comment:
This looks like it returns a tuple to me, not a dict: https://github.com/pika/pika/blob/e1f7c83ad2279ca98d33151e59955262461dd509/pika/data.py#L198